exercises/binary-search/src/test/scala/BinarySearchTest.scala

```diff
@@ -0,0 +1,74 @@
+import org.scalatest.{FunSuite, Matchers}
+
+class BinarySearchTest extends FunSuite with Matchers {
+  test("finds a value in an array with one element") {
+    pending
+    val elements = Array[Int](6)
+    val result = BinarySearch.search(elements, 6)
+    assert(result.contains(0))
```

Comment:
Why not be more explicit about the return type as in the other exercises?
```scala
result should be (Some(0))
```
(same for the other test cases).